### ISSUE TITLE
Shutdown clients in parallel

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud-docker#576]
### How it is

`down` triggers shutdown of clients in sequence. If one or more clients don't shutdown cleanly and have to be killed by the docker 3 minute timeout, this can add add up to a 10minute+ shutdown time

### How it should be

`down` triggers shutdown of clients in parallel

**Comment by kilrau** [comment]
Closing in favor of https://github.com/ExchangeUnion/xud-docker/issues/701
